widgetssdk/src/main/java/com/glia/widgets/call/CallState.java

```diff
@@ -328,16 +331,28 @@ public CallState setTransferring() {
                 .createCallState();
     }
 
-    public boolean isMuteButtonEnabled() {
-        return (isAudioCall() || isVideoCall()) && !showOnHold();
+    public ViewState getGetMuteButtonViewState() {
+        if (isCallVisualizer) {
+            return ViewState.HIDE;
+        } else if ((isAudioCall() || isVideoCall()) && !showOnHold()) {
+            return ViewState.SHOW;
+        } else {
+            return ViewState.DISABLE;
+        }
     }
 
     public boolean isVideoButtonEnabled() {
         return is2WayVideoCall() && !showOnHold();
     }
 
-    public boolean isSpeakerButtonEnabled() {
-        return isAudioCall() || isVideoCall();
+    public ViewState getGetSpeakerButtonViewState() {
```

Comment:
Looks like "Get" is unnecessary here, isn't it?
`getGetSpeakerButtonViewState()` => `getSpeakerButtonViewState()`